bin/mail/2016-08-09-hoc-latam/send-real-emails

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+#3661
+../send-to-mailing-list --limit 4000 08-8-16-latam-hoc .latam-es-contacts.csv
+
+#3170
+../send-to-mailing-list --limit 4000 08-8-16-Brazil-hoc .latam-pt-contacts.csv
```

Comment:
Jon created the templates and supplied those names. I'll ask him if it's trouble to rename.